package/debian/make_deb.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+VER_NUM=$(echo $TRAVIS_TAG | cut -d 'v' -f 2) # extract version number
+TIMESTAMP=$(date "+%a, %d %b %Y %H:%M:%S %z")
+
+cat << EOF > package/debian/debian/changelog
+gittown ($VER_NUM) RELEASED; urgency=low
+
+  * See https://github.com/Originate/git-town/releases for a complete changelog
+
+ -- allonsy <alec.snyder@originate.com> $TIMESTAMP
+EOF
+
+cp dist/git-town-linux-amd64 package/debian/git-town
+cd package/debian
+sudo apt-get -qq update
+sudo apt-get install -y build-essential debhelper devscripts
+./debian_build.sh
+cd ../..
+cp package/gittown_"$VER_NUM"_amd64.deb dist
```

Comment:
We don't currently have the version number in the executables on the releases page. Thoughts on removing the version number? Otherwise I think we should update the executables to also have the version number

Reply:
Since it is already attached to the releases page, I think it is OK to remove the release number. I've taken a look at some other go projects on github and they don't include the version number either so it would seem that we should be in good company if we remove it.
I'll push up a change soon that suppresses the version number.